Source/VungleAdapterBannerAd.swift

```diff
@@ -159,6 +147,9 @@ final class VungleAdapterBannerAd: VungleAdapterAd, PartnerAd {
             // Non-programmatic
             try VungleSDK.shared().addAdView(to: inlineView, withOptions: [:], placementID: request.partnerPlacement)
         }
+        
+        // Mark ad as displayed so we know if we need to mark it as finished later on `invalidate()`.
+        adWasDisplayed = true
```

Comment:
`invalidate()` does not set `adWasDisplayed` back to `false`. Is it expected behavior?

```
func invalidate() throws {
        if adWasDisplayed {
            finishDisplayingVungleAd()
        }
        log(.invalidateSucceeded)
    }
```

Reply:
yeah, invalidate is called when the ad is discarded, just before it is deallocated. So there's no need to clean up state at that point